metrics_block.py

```diff
@@ -54,6 +65,56 @@ def stop(self):
         if self._metrics_job is not None:
             self._metrics_job.cancel()
 
+    def timestamp(self):
+        '''returns the current system timestamp'''
+        return datetime.isoformat(datetime.utcnow())
+
+    @staticmethod
```

Comment:
Ohhh it's because it doesn't have a self in the arguments list, that's why your editor is throwing that at you.

I think add the `self` remove the `staticmethod`